app/components/Views/SDKSessionsManager/SDKSessionsManager.tsx

```diff
@@ -83,9 +86,13 @@ const SDKSessionsManager = (props: Props) => {
   };
 
   useEffect(() => {
-    const refreshSDKState = () => {
+    const refreshSDKState = async () => {
       const _connections = sdk.getConnections();
+      const _androidConnections = await sdk.loadAndroidConnections();
       const connectionsList = Object.values(_connections);
+      const androidConnectionsList = Object.values(_androidConnections);
+
+      setAndroidConnections(androidConnectionsList);
```

Comment:
Consider adding error handling around this asynchronous call to manage potential failures.

```suggestion
      try {
        const _androidConnections = await sdk.loadAndroidConnections();
        const androidConnectionsList = Object.values(_androidConnections);
        setAndroidConnections(androidConnectionsList);
      } catch (error) {
        console.error("Failed to load Android connections:", error);
      }
```